packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -604,7 +604,7 @@ pub async fn load_next_config_internal(
     )
     .await?;
 
-    let SingleValue::Single(Ok(val)) = config_value.into_single().await else {
+    let SingleValue::Single(val) = config_value.try_into_single().await.context("Evaluation of Next.js config failed")? else {
```

Comment:
Nit: long line. Rust formatted can't handle let-else for some reason.

Reply:
I'll do that in a separate PR...